index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio</title>
+    <title>Portfolio Main Page</title>
```

Comment:
- [ ] Kindly make sure to update the title of your portfolio in order to make it professional and adequately and briefly describes the content of the page. For example:

    ### Poor titles:

     - Microverse project | Your portfolio.
     - Portifolio project


 ### Better page titles:

   - Mark's portfolio
   - Mark full-stack web developer